docs/upgrade.html

```diff
@@ -45,6 +45,22 @@ <h5><a id="upgrade_240_notable" href="#upgrade_240_notable">Notable changes in 2
         The old overloaded functions are deprecated and we would recommend users to make their code changes to leverage the new methods (details
         can be found in <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-520%3A+Add+overloaded+Consumer%23committed+for+batching+partitions">KIP-520</a>).
     </li>
+    <li>We are introducing incremental cooperative rebalancing to the clients' group protocol, which allows consumers to keep all of their assigned partitions during a rebalance
+        and at the end revoke only those which must be migrated to another consumer for overall cluster balance. To upgrade from eager to cooperative rebalancing, you must set all your clients
```

Comment:
.. from the old (eager) protocol that always revoke all partitions before rebalancing to cooperative rebalancing ..